echopype/calibrate/api.py

```diff
@@ -49,29 +49,36 @@ def _compute_cal(
         waveform_mode=waveform_mode,
         encode_mode=encode_mode,
     )
-    # Perform calibration
-    if cal_type == "Sv":
-        sv_dataset = cal_obj.compute_Sv(waveform_mode=waveform_mode, encode_mode=encode_mode)
-        # Add attributes
-        sv_dataset["Sv"].attrs = {
-            "long_name": "Volume backscattering strength (Sv)",
-            "units": "dB re 1 m-1",
+
+    def add_attrs(cal_type, ds):
+        """Add attributes to backscattering strength dataset.
+        cal_type: Sv or TS
+        """
+        ds["range_sample"].attrs = {"long_name": "Along-range sample number, base 0"}
+        ds["echo_range"].attrs = {"long_name": "Range distance", "units": "m"}
+        ds[cal_type].attrs = {
+            "long_name": {
+                "Sv": "Volume backscattering strength (Sv re 1 m-1)",
+                "TS": "Target strength (TS re 1 m^2)",
+            }[cal_type],
+            "units": "dB",
             "actual_range": [
-                round(float(sv_dataset["Sv"].min().values), 2),
-                round(float(sv_dataset["Sv"].max().values), 2),
+                round(float(ds[cal_type].min().values), 2),
+                round(float(ds[cal_type].max().values), 2),
             ],
         }
-        sv_dataset["range_sample"].attrs = {"long_name": "Along-range bin (sample) number, base 0"}
-        sv_dataset["echo_range"].attrs = {"long_name": "Range distance", "units": "m"}
 
+    # Perform calibration
+    if cal_type == "Sv":
+        sv_dataset = cal_obj.compute_Sv(waveform_mode=waveform_mode, encode_mode=encode_mode)
+        add_attrs("Sv", sv_dataset)
         if "water_level" in echodata.platform.data_vars.keys():
             # add water_level to the created xr.Dataset
             sv_dataset["water_level"] = echodata.platform.water_level
```

Comment:
@emiliom : Not sure if this should be in another PR: 

Looking at this now, I realized the check and addition of `water_level` should be in the outputs for both compute_Sv and compute_TS so that users can use this to convert `echo_range` to `depth` easily. 

Also in the realm of this PR, the attributes of `water_level` can be set in `_set_MVBS_attrs` below under echopype/preprocess/api.py.

Reply:
Related to #516 #583 #586

Reply:
It could be a separate PR. But this looks like an easy change w/o additional ramifications that will lead to protracted discussions. It's also in line with the broader improvements to `compute_MVBS_index_binning`. So, I'll just add it to this PR.

Reply:
Sounds good. Thank you @emiliom !

Reply:
> Also in the realm of this PR, the attributes of water_level can be set in _set_MVBS_attrs below under echopype/preprocess/api.py.

Actually, `water_level` retains its original attributes from the echodata `Platform` group. So, no attribute assignment is taking place in `calibrate/api.py`.

I've implemented the addition of `water_level` to `compute_TS` (and in the process it made the code quite a bit simpler!). I'll push it to the PR in a bit.